Source/ACE.Entity/Position.cs

```diff
@@ -10,7 +10,17 @@
     [DbGetList("vw_character_positions", 23, "character_id")]
     public class Position
     {
-        public LandblockId LandblockId { get; set; }
+        public LandblockId LandblockId
+        {
+            get
+            {
+                return new LandblockId(Cell);

```

Comment:
This doesn't feel helpful.  Essentially re-allocating a new LandblockId every time it is accessed.  I agree the section below in the constructor was a bug, but I don't see the bug in this part.